ui/src/models/index.ts

```diff
@@ -150,9 +150,16 @@ export function getNextSchedule(data: WorkflowListItem): number {
   if (!schedules || schedules.length == 0 || data.Suspended) {
     return Number.MAX_SAFE_INTEGER;
   }
-  const datesToRun = schedules.map((s) =>
-    cronParser.parseExpression(s.Expression).next()
-  );
+  const tz = getConfig().timeZone;
```

Comment:
For backwards compatibility tz won't be set, so it should defer to the original behaviour.